typed-racket-lib/typed-racket/base-env/base-env.rkt

```diff
@@ -696,6 +696,11 @@
                  (cl->* ((-lst a) Univ . -> . (-opt -Index))
                         ((-lst a) b (-> a b Univ)
                            . -> . (-opt -Index))))]
+[index-where (-poly (a b)
+                    (Univ (-lst a)
```

Comment:
What is the first argument of type Univ for?

Reply:
oops, not necessary